Prepare for release 0.26.1

kschoche opened the pull request:
Fixes a bug preventing templating of `server.ha.replicas`.

```
+## 0.26.1 (October 30, 2023)
+
+Bugs:
+* Fix templating of `server.ha.replicas` when set via override file. [GH-961](https://github.com/hashicorp/vault-helm/pull/961)
+
```

swenson commented on CHANGELOG.md:
```diff
@@ -1,7 +1,16 @@
 
+KNOWN ISSUES:
+
+* The chart will ignore `server.ha.replicas` and always deploy 3 server replicas when `server.ha.enabled=true` unless overridden by command line when issuing the helm command: `--set server.ha.replicas=<some_number>`. Fixed in [GH-961](https://github.com/hashicorp/vault-helm/pull/961)
```
I'm not sure it is necessary to put this here if we are fixing it in 0.26.1?

swenson reviewed (approved):
LGTM, one question

swenson commented:
I'm okay either way. But presumably all bug fixes were known issues in 1 or more prior releases :)

kschoche commented:
> I'm okay either way. But presumably all bug fixes were known issues in 1 or more prior releases :)

Thanks, good call. I removed the known issues section and moved some of its contents in support of the "fix".
